MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -14,25 +14,38 @@ internal class MiniAppWebView: WKWebView {
 
     convenience init(miniAppId: String, versionId: String, queryParams: String? = nil) {
         let schemeName = Constants.miniAppSchemePrefix + miniAppId
-        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName + MiniAppWebView.getQueryParams(queryParams: queryParams))!)
         let config = MiniAppWebView.defaultConfig()
         config.setURLSchemeHandler(URLSchemeHandler(versionId: versionId), forURLScheme: schemeName)
         self.init(frame: .zero, configuration: config)
-        commonInit(urlRequest: urlRequest)
+        commonInit(urlRequest: Self.getURLRequest(miniAppId: miniAppId, schemeName: schemeName, queryParams: queryParams))
     }
 
     convenience init(miniAppURL: URL, queryParams: String? = nil) {
-        let urlWithQueryParam = miniAppURL.appendingPathComponent(MiniAppWebView.getQueryParams(queryParams: queryParams))
-        let urlRequest = URLRequest(url: urlWithQueryParam)
+        let urlRequest = URLRequest(url: Self.getLoadByURLRequest(miniAppURL: miniAppURL, queryParams: queryParams))
         self.init(frame: .zero, configuration: MiniAppWebView.defaultConfig())
         commonInit(urlRequest: urlRequest)
     }
 
+    private static func getLoadByURLRequest(miniAppURL: URL, queryParams: String?) -> URL {
```

Comment:
I suggest we rename this getURL(_ miniAppURL: URL, with queryParameters: String?)

Reply:
Done